packages/react/src/components/form/flow.ts

```diff
@@ -42,53 +49,16 @@ export interface ErrorState {
   cancel: Cancel;
 }
 
-interface LoginEvent {
-  type: "sid_login";
-  config: LoginConfiguration;
-  options?: LoginOptions;
-}
-
-interface LoginSuccessEvent {
-  type: "sid_login.success";
-  user: User;
-}
-
-interface LoginErrorEvent {
-  type: "sid_login.error";
-  error: Error;
-}
-
-interface CancelEvent {
-  type: "sid_cancel";
-}
-
-interface RetryEvent {
-  type: "sid_retry";
-  context: AuthenticatingState["context"];
-}
-
-interface InitEvent {
-  type: "sid_init";
-}
-
-type Event =
-  | InitEvent
-  | LoginEvent
-  | LoginSuccessEvent
-  | LoginErrorEvent
-  | RetryEvent
-  | CancelEvent;
-
-type FlowActions = {
+export type FlowActions = {
   // a function that will be called when the state is entered
   entry?: () => void;
 };
 
 export type FlowState = FlowActions &
   (InitialState | AuthenticatingState | SuccessState | ErrorState);
 
-type Observer = (state: FlowState, event: Event) => void;
-type Send = (e: Event) => void;
+export type Observer = (state: FlowState, event: Event) => void;
```

Comment:
Should this be moved to the dedicated types module?

Reply:
https://github.com/slashid/javascript/pull/239/commits/e6a560b405604d99c62f7ffbb12ad62e2cd06b50